lms/djangoapps/course_goals/migrations/0007_set_unsubscribe_token_default.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.24 on 2021-08-20 19:18
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
```

Comment:
This second migration is to provide `default=uuid.uuid4` for future rows. Existing rows will get a `null` value from the 0006 migration. (To backfill uuids, we'd need to write custom python, since mysql can't do that automatically.)